tox.ini

```diff
@@ -1,9 +1,19 @@
-[pep8]
-ignore = E226,E302,E41,E251,C901
-max-line-length = 160
+[tox]
+minversion = 1.6
+skipsdist = True
+envlist = pep8
+
+[testenv]
+install_command = pip install -U {opts} {packages}
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
+
+[testenv:pep8]
+commands =
+    flake8
 
 [flake8]
 ignore = E226,E302,E41,E251,C901
 max-line-length = 160
```

Comment:
I think the max-line-length should be a bit smaller, 100 ?

@goern @vpavlin : thoughts?